MastodonSDK/Sources/MastodonSDK/MastodonFeed.swift

```diff
@@ -0,0 +1,75 @@
+// Copyright © 2023 Mastodon gGmbH. All rights reserved.
+
+import Foundation
+import CoreDataStack
+
+public final class MastodonFeed {
```

Comment:
`MastodonFeed` replaces the original `Feed` item which is an abstraction over features such as loading gaps in between statuses. I _think_ we should be able to get rid of it, but I didn't dare to do so yet because it would have resulted in an even bigger set of changes.